packages/framework/inline/src/services/range.ts

```diff
@@ -73,9 +74,10 @@ export class RangeService<TextAttributes extends BaseTextAttributes> {
       // range change may happen before the editor is prepared
       await this.editor.waitForUpdate();
       // improve performance
-      requestIdleCallback(() => {
-        this.editor.requestUpdate(false);
-      });
+      REQUEST_IDLE_CALLBACK_ENABLED &&
```

Comment:
If requestIdleCallback is not supported, maybe we should call this with setTimeout or Promise.resolve. @Flrande what do you think?

Reply:
`Promise.resolve` look good to me

Reply:
Change to:

```ts
      if (REQUEST_IDLE_CALLBACK_ENABLED) {
        requestIdleCallback(() => {
          this.editor.requestUpdate(false);
        });
      } else {
        await Promise.resolve(this.editor.requestUpdate(false));
      }
```